proposals/runtime_get_contexts.md

```diff
@@ -236,6 +238,22 @@ This is an artifact of existing APIs and precedence. Since many existing APIs
 use the constant integer values, we want to be consistent with those. However,
 for newly-introduced fields, we use the more intuitive `undefined` state.
 
+
+### Developer Tools Contexts
+
+Extensions can use the
+[Dev Tools API](https://developer.chrome.com/docs/extensions/mv3/devtools/) to
```

Comment:
Using https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools might be slightly more browser neutral, but I don't have strong feelings.

Reply:
Agreed. Changed the link.